src/components/NavigationModal/utils/Books.tsx

```diff
@@ -1,26 +1,22 @@
+import Box from "@mui/material/Box";
 import Grid from "@mui/material/Grid";
+import { useEffect, useRef, useState } from "react";
 import { books as newTestamentMetadata } from "../../../applicationLogic/data/newTestamentMetadata";
 import { oldTestamentBooks } from "../../../applicationLogic/data/oldTestamentMetadata";
-import { useEffect, useRef, useState } from "react";
-import BookSearchBar from "../../BookSearchBar";
-import Box from "@mui/material/Box";
 import { BOOKCHAPTERMENU_OFFSET } from "../../../constants";
+import BookSearchBar from "../../BookSearchBar";
 
 interface BookProps {
 	bookData: any[];
 	handleClick: (params: any) => any;
 	isCurrentBook: boolean;
 }
 
-function Book({
-	bookData,
-	handleClick,
-	isCurrentBook,
-}: BookProps) {
+function Book({ bookData, handleClick, isCurrentBook }: BookProps) {
 	const bookRef = useRef<null | HTMLDivElement>(null);
 
 	useEffect(() => {
-		if (isCurrentBook == true) {
+		if (isCurrentBook === true) {
```

Comment:
A little more idiomatic JS is to usually just do something like if (!!isCurrentBook) as a general principle. This is fine as is, but it's just a little more explicity.